### ISSUE TITLE
Removes entire `sinon.stub` call if stub function is not declared inline.

### ISSUE THREAD

**Issue by transplier** [hurrymaplelad/sinon-codemod#8]
`test.js`:
```js
const stubFunction = () => {};
const fooStub = sinon.stub({}, 'foo', stubFunction);
```
```sh
jscodeshift -t extract-calls-fake.js test.js
```
Produces:
```js
const stubFunction = () => {};
const fooStub;
```

As you can see, the call to sinon is completely removed.

**Comment by hurrymaplelad** [comment]
Well that's not good! PR welcome!

**Comment by hurrymaplelad** [comment]
May be addressed by #7, looking...

**Comment by hurrymaplelad** [comment]
Fixed by #7.
